plugins/shortcuts/src/EditShortcut.test.tsx

```diff
@@ -132,8 +169,14 @@ describe('EditShortcut', () => {
 
     await renderInTestApp(
       <>
-        <AlertDisplay />
-        <EditShortcut {...props} />
+        <TestApiProvider
+          apis={[
+            [shortcutsApiRef, new DefaultShortcutsApi(MockStorageApi.create())],
+          ]}
+        >
+          <AlertDisplay />
+          <EditShortcut {...props} />
+        </TestApiProvider>
```

Comment:
I guess some new tests are missing here.

However, shouldn't the following be valid? 🤔 

```js
[
  {
    title: "A",
    url: "/foo"
  },
  {
    title: "Z",
    url: "/foo"
  }
]  
```

Reply:
Agreed, a test that after trying to add a new conflicting shortcut, or renaming an old shortcut to have a conflicting name, you actually get the right notification.